### ISSUE TITLE
v4: generated stylesheet can have invalid JSON

### ISSUE THREAD

**Issue by johnf** [marklawlor/nativewind#599]
**Describe the bug**
I used `border-none` as a class, this is incorrect, it should be `border-0`
I see the following in the console:
```
 ERROR  SyntaxError: JSON Parse error: Unexpected character: o, js engine: hermes
 LOG  Running "Gladly" with {"rootTag":11}
 ERROR  Invariant Violation: "Gladly" has not been registered. This can happen if:
* Metro (the local dev server) is run from the wrong folder. Check if Metro is running, stop it and restart it in the current project.
* A module failed to load due to an error and `AppRegistry.registerComponent` wasn't called., js engine: hermes
```

**To Reproduce**
1. Add border-none as a class

**Expected behavior**
Prints invalid property error

**Additional context**
The stylesheet being passed to JSON.parse has this section
```json
    "border-none": {
      "style": {},
      "specificity": {
        "A": 0,
        "B": 1,
        "C": 0,
        "I": 0,
        "S": 1,
        "O": 71
      },
      "warnings": [
        {
          "type": "IncompatibleNativeValue",
          "property": "border-style",
          "value": ""none""
        }
      ]
```

notice the double quotes around `none`

I tried tracking this down and I think native wind is doing the right thing. It's object looks like
```js
{
  type: 'IncompatibleNativeValue',
  property: 'border-style',
  value: '"none"'
}
```

So maybe this is a bug in lighting.css?
```

and my suspicion is that this is probably a bug in lightning.css

**Comment by johnf** [comment]
I also wonder if it's worth adding some catch statements in https://github.com/marklawlor/nativewind/blob/main/packages/nativewind/src/metro/transformer.ts#L36-L39
that can print some errors and register an empty stylesheet
It took me a while to track this down since all you know is that JSON somewhere is bad

**Comment by marklawlor** [comment]
This is a bug the error handling of `border-style` in our parser (it was adding the extra quotes). I'll release it later today.

Unfortunately to support native you need to use `border-0` instead of `border-none`. I don't think there is a way for NativeWind to correctly implement `border-none` without hitting edge cases e.g `!border-1 border-none` will render differently on web/native. We'll have to wait for [RN to implement it](https://github.com/facebook/react-native/issues/34425). I should add this to the documentation.

**Comment by marklawlor** [comment]
Fixed in `4.0.1`. I've switched away from pre-release versions so people can use `^` in their `package.json` to get updates.
